components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -149,6 +149,15 @@ pub struct RequestSplit {
     pub source: Cow<'static, str>,
 }
 
+#[derive(Debug)]
+pub struct RequestHalfSplit {
```

Comment:
If it's test only, it would be better to make it test only. Check other example for test-only structures (search testexport)

Reply:
ditto